.gitignore

```diff
@@ -24,9 +24,9 @@ dist-ssr
 *.sw?
 
 # lock files
-package-lock.json
-yarn.lock
-pnpm-lock.yaml
+# package-lock.json
+# yarn.lock
+# pnpm-lock.yaml
 

```

Comment:
I understand you whant to push an lock config, but do you realy whant to have every type of lock config in your project?
(pnpm, npm, yarn,...)